pkg/injector/webhook.go

```diff
@@ -162,7 +212,7 @@ func (wh *webhook) mutateHandler(w http.ResponseWriter, req *http.Request) {
 		log.Error().Err(err).Msg("Error decoding admission request")
 		admissionResp.Response = toAdmissionError(err)
 	} else {
-		admissionResp.Response = wh.mutate(admissionReq.Request)
+		admissionResp.Response, webhookResourceName = wh.mutate(admissionReq.Request)
```

Comment:
where is `webhookResourceName` being used?

Reply:
it's read and used by the defer function. Used to log what resource took how much time.